odoo/cli/tsconfig.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import glob
+import itertools
+import os
+import sys
+import re
+import json
+
+from . import Command
+from odoo.modules.module import MANIFEST_NAMES
+
+
+class TSConfig(Command):
+    """Generates tsconfig files for javascript code"""
+
+    def __init__(self):
+        self.command_name = "tsconfig"
```

Comment:
Seems unnecessary, the runner should atomatically set the lowercased class name as `name` on the class.